test/spoom/cli/bump_test.rb

```diff
@@ -367,6 +367,29 @@ class A; end
         assert_equal("true", Sorbet::Sigils.file_strictness("#{@project.path}/file1.rb"))
         assert_equal("false", Sorbet::Sigils.file_strictness("#{@project.path}/vendor/file2.rb"))
       end
+
+      def test_bump_count_errors
+        @project.write("file1.rb", <<~RB)
+          # typed: false
+          class Foo
+            def foo
+            end
+          end
+
+          Foo.new.foos
+        RB
+
+        out, err, status = @project.bundle_exec("spoom bump --no-color --count-errors")
+        assert_empty(err)
+        assert_equal(<<~OUT, out)
+          Checking files...
+
+          Errors: 1
```

Comment:
Reading this it looks like the errors are about the `bump` itself.

What do you think about something like "Found X type checking error(s)."?